csss-site/src/events/templates/events/fall_hacks_submissions2020.html

```diff
@@ -0,0 +1,329 @@
+{% extends "csss/new_header.html" %}
+
+{% block content %}
+{{ block.super }}
+    <title>Fall Hacks 2020</title>

```

Comment:
you sure this is correct? when I load the page at https://dev.sfucsss.org/PR-280/events/fall_hacks_submissions2020 , I see "SFU CSSS" on the tab.

Reply:
seems the issue is that there is a title in the template, and first title is the one that makes a difference. will remove it.